<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_Order">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>订</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>3</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{border:1px solid rgb(220,220,220)}</string>
   </property>
   <widget class="QLabel" name="txt_DepartTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>09:15</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_ArriveTime">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>13:25</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>3</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>4h10m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txt_Depart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>首都机场 T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_Arrive">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>37</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>地窝堡机场 T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_craftType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>76</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CA1253</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="txt_price">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>10</y>
     <width>69</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1940元</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
